src/components/bar.cpp

```diff
@@ -415,19 +415,11 @@ void bar::parse(string&& data, bool force) {
 
   m_renderer->end();
 
-  const auto check_dblclicks = [&]() -> bool {
-    if (m_action_ctxt->has_double_click()) {
-      return true;
-    }
-
-    for (auto&& action : m_opts.actions) {
-      if (static_cast<int>(action.button) >= static_cast<int>(mousebtn::DOUBLE_LEFT)) {
-        return true;
-      }
+  for (auto&& action : m_opts.actions) {
```

Comment:
Note to self: Global bar click actions should somehow be integrated into the action context at some point. It would make a lot of code here cleaner.